### PR TITLE
fix: (docs) modify space and chinese/english comma

### PR DESCRIPTION
**Pull request by Derekmini**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

<!--
请在 `routes` 区域填写以 / 开头的完整路由地址，否则你的 PR 将会被无条件关闭。
如果路由包含在文档中列出可以完全穷举的参数（例如分类），请依次全部列出。

Please include route starts with /, with all required and optional parameters. Fail to comply will result in your pull request being closed automatically.
```routes
/some/route
/some/other/route
如果你的 PR 与路由无关, 请在 route 区域 填写 `NOROUTE`，而不是直接删除 `routes` 区域。否则你的 PR 将会被无条件关闭。
If your changes are not related to route, please fill in `routes` with `NOROUTE`. Fail to comply will result in your PR being closed.
-->

```routes
NOROUTE
```

## 新RSS检查列表 / New RSS Script Checklist
  
- [ ] New Route
- [ ] Documentation
  - [ ] CN
  - [ ] EN
- [ ] 全文获取 fulltext
  - [ ] Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] 日期和时间 date and time
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added 
- [ ] `Puppeteer`

## 说明 / Note

修改 @TonyRL 提出的修改内容

### CODE REVIEW

File: docs/journal.md [hunk 1]
```diff
@@ -108,13 +108,15 @@ pageClass: routes
 
 ### 最新文章
 
-<Route author="Derekmini auto-bot-ty" example="/ieee/70/latest/vol" path="/:journal/latest/date/:sortType?" :paramsDesc="['期刊名称， 复制 URL 中 punumber 部分','排序方式, 默认`vol-only-seq`，复制 URL 中 sortType 部分']" radar="1" rssbud="1">
+<Route author="Derekmini auto-bot-ty" example="/ieee/70/latest/vol" path="/ieee/:journal/latest/date/:sortType?" :paramsDesc="['期刊名称，复制 URL 中 punumber 部分','排序方式，默认`vol-only-seq`，复制 URL 中 sortType 部分']" radar="1" rssbud="1">
```
Comment (TonyRL) [inline docs/journal.md:111]:
```suggestion
<Route author="Derekmini auto-bot-ty" example="/ieee/70/latest/vol" path="/ieee/:journal/latest/vol/:sortType?" :paramsDesc="['期刊名称，复制 URL 中 punumber 部分','排序方式，默认`vol-only-seq`，复制 URL 中 sortType 部分']" radar="1" rssbud="1">
```

File: docs/en/journal.md [hunk 2]
```diff
@@ -104,23 +104,23 @@ Google Scholar has strict anti-crawling mechanism implemented, the demo below do
...
+The parameter id in the route is the id in the URL of the user's Google Scholar reference page, for example `https://scholar.google.com/citations?hl=zh-CN&user=mlmE4JMAAAAJ` to `mlmE4JMAAAAJ`
 
 </RouteEn>
 
 ## IEEE Xplore
 
 ### Latest Research
 
-<RouteEn author="Derekmini auto-bot-ty" example="/ieee/70/latest/vol" path="/:journal/latest/date/:sortType?" :paramsDesc="['Journal Name， get it from punumber of the URL','Sort Type, default: `vol-only-seq`，get it from sortType of the URL']" radar="1" rssbud="1">
+<RouteEn author="Derekmini auto-bot-ty" example="/ieee/70/latest/vol" path="/:journal/latest/date/:sortType?" :paramsDesc="['Journal Name, get it from punumber of the URL','Sort Type, default: `vol-only-seq`, get it from sortType of the URL']" radar="1" rssbud="1">
```
Comment (TonyRL) [inline docs/en/journal.md:115]:
```suggestion
<RouteEn author="Derekmini auto-bot-ty" example="/ieee/70/latest/vol" path="/:journal/latest/vol/:sortType?" :paramsDesc="['Journal Name, get it from punumber of the URL','Sort Type, default: `vol-only-seq`, get it from sortType of the URL']" radar="1" rssbud="1">
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: docs/en/journal.md (hunk 2 above)
Comment (Derekmini) [reply docs/en/journal.md]: Sorry about that, I have paid attention to this mistake in a wrong part, but I didn't realize that until now.

Thanks for your attention!!!

File: docs/en/journal.md [hunk 3]
```diff
@@ -104,23 +104,23 @@ Google Scholar has strict anti-crawling mechanism implemented, the demo below do
...
 
-<RouteEn author="Derekmini auto-bot-ty" example="/ieee/70/latest/vol" path="/:journal/latest/date/:sortType?" :paramsDesc="['Journal Name， get it from punumber of the URL','Sort Type, default: `vol-only-seq`，get it from sortType of the URL']" radar="1" rssbud="1">
+<RouteEn author="Derekmini auto-bot-ty" example="/ieee/70/latest/vol" path="/:journal/latest/vol/:sortType?" :paramsDesc="['Journal Name, get it from punumber of the URL','Sort Type, default: `vol-only-seq`, get it from sortType of the URL']" radar="1" rssbud="1">
 
 </RouteEn>
 
 ### Latest Research (Last 2 month)
 
-<RouteEn author="Derekmini auto-bot-ty" example="/ieee/78/latest/date" path="/:journal/latest/date/:sortType?" :paramsDesc="['Journal Name， get it from punumber of the URL','Sort Type, default: `vol-only-seq`，get it from sortType of the URL']" radar="1" rssbud="1">
+<RouteEn author="Derekmini auto-bot-ty" example="/ieee/78/latest/date" path="/:journal/latest/date/:sortType?" :paramsDesc="['Journal Name, get it from punumber of the URL','Sort Type, default: `vol-only-seq`, get it from sortType of the URL']" radar="1" rssbud="1">
```
Comment (TonyRL) [inline docs/en/journal.md:121]:
```suggestion
<RouteEn author="Derekmini auto-bot-ty" example="/ieee/78/latest/date" path="/ieee/:journal/latest/date/:sortType?" :paramsDesc="['Journal Name, get it from punumber of the URL','Sort Type, default: `vol-only-seq`, get it from sortType of the URL']" radar="1" rssbud="1">
```

_Originally posted by @TonyRL in https://github.com/DIYgod/RSSHub/pull/8584#discussion_r757771773_

Comment (Derekmini) [reply docs/en/journal.md:121]: Thanks for your attention ! I have modified it and will get a new pr.

**Pull request merged**